vtex-apps/shelf
# Fix/cms sort options

#### What is the purpose of this pull request?
It is now possible to order the shelf by more criteria and the default sorting is by relevance.

#### What problem is this solving?
Before this PR the only ordering methods were by sales and price, and the default sorting algorithm was by sales.

#### How should this be manually tested?
Add a new shelf to the `blocks.json` of the store theme, open the storefront and check the new shelf's `List Ordenation` value and the possible values to this field.

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/8443580/61733129-7d729f00-ad55-11e9-8817-10ca2f056660.png)

#### Types of changes
